app/models/attendee.js

```diff
@@ -40,5 +41,25 @@ export default ModelBase.extend({
   event  : belongsTo('event'),
   order  : belongsTo('order'),
   ticket : belongsTo('ticket'),
-  user   : belongsTo('user')
+  user   : belongsTo('user'),
+
+  ticketPrice: computed('ticket.price', 'ticket.discountCodes', 'order.discountCode', function() {
```

Comment:
also watch `order`, `ticket`, `ticket.discountCodes.@each`

Reply:
@CosmicCoder96 the ticket prices being returned are `price` in every case. Even when using async await, the FE displays 0 before any data is being returned. ANy better approach?